src/Microsoft.DotNet.Build.Tasks/project.json

```diff
@@ -4,14 +4,14 @@
     "Microsoft.Build.Framework": "0.1.0-preview-00005",
     "Microsoft.Build.Tasks.Core": "0.1.0-preview-00005",
     "Microsoft.Build.Utilities.Core": "0.1.0-preview-00005",
-    "Microsoft.CSharp": "4.0.0",
-    "System.Linq": "4.0.0",
-    "System.Xml.XDocument": "4.0.0",
-    "System.IO.Compression": "4.0.0",
-    "System.IO.Compression.ZipFile": "4.0.0",
-    "System.Xml.ReaderWriter": "4.0.10",
+    "Microsoft.CSharp": "4.0.1-beta-23328",
```

Comment:
I forgot to remove this change. Basically when we moved all dependencies up to tool-runtime/project.json I had to take the later version of each dependency in all project.jsons, which was a beta. After that, Eric suggested to clean up tool-runtime/project.json since I was referencing a meta package already, so it wasn't necesary to have dependencies to all the betas so I removed it from there, but didn't remove it from here. I'll fix it.